### PR TITLE
Docs: vault-helm 0.23.0 updates

### PR DESCRIPTION
**Pull request by tomhjp**
Updates website documentation for v0.23.0 release.

See https://github.com/hashicorp/vault-helm/releases/tag/v0.23.0 for changelog.

### CODE REVIEW

File: website/content/docs/platform/k8s/helm/configuration.mdx [hunk 1]
```diff
@@ -648,6 +648,14 @@ and consider if they're appropriate for your deployment.
 
     - `enabled` (`boolean: true`) - When set to `true`, a Kubernetes service will be created for Vault.
 
+    - `active` - Values that apply only to the vault-active service.
+
+      `enabled` (`boolean: true`) - When set to `true`, the vault-active Kubernetes service will be created for Vault, selecting pods which label themselves as the cluster leader with `vault-active: "true"`.
```
Comment (tvoran) [inline website/content/docs/platform/k8s/helm/configuration.mdx:653]: It looks like this should be a separate bullet under `active`?

```suggestion
      - `enabled` (`boolean: true`) - When set to `true`, the vault-active Kubernetes service will be created for Vault, selecting pods which label themselves as the cluster leader with `vault-active: "true"`.
```

File: website/content/docs/platform/k8s/helm/configuration.mdx [hunk 2]
```diff
@@ -648,6 +648,14 @@ and consider if they're appropriate for your deployment.
...
+
+    - `standby` - Values that apply only to the vault-standby service.
+
+      `enabled` (`boolean: true`) - When set to `true`, the vault-standby Kubernetes service will be created for Vault, selecting pods which label themselves as a cluster follower with `vault-active: "false"`.
```
Comment (tvoran) [inline website/content/docs/platform/k8s/helm/configuration.mdx:657]: Same question here: 

```suggestion
      - `enabled` (`boolean: true`) - When set to `true`, the vault-standby Kubernetes service will be created for Vault, selecting pods which label themselves as a cluster follower with `vault-active: "false"`.
```

File: website/content/docs/platform/k8s/helm/configuration.mdx [hunk 3]
```diff
@@ -666,6 +674,10 @@ and consider if they're appropriate for your deployment.
 
     - `publishNotReadyAddresses` (`boolean: true`) - If true, do not wait for server pods to be ready before adding them to the service pool.
 
+    - `instanceSelector`
+
+      `enabled` (`boolean: true`) - When set to false, the service selector used for the vault, vault-active, and vault-standby services will not filter on `app.kubernetes.io/instances`. This means they may select pods from outside this deployment of the Helm chart. Does not affect the headless vault-internal service with `ClusterIP: None`.
```
Comment (tvoran) [inline website/content/docs/platform/k8s/helm/configuration.mdx:679]:
```suggestion
      - `enabled` (`boolean: true`) - When set to false, the service selector used for the vault, vault-active, and vault-standby services will not filter on `app.kubernetes.io/instances`. This means they may select pods from outside this deployment of the Helm chart. Does not affect the headless vault-internal service with `ClusterIP: None`.
```

**Review by tvoran — APPROVED** [review]

File: website/content/docs/platform/k8s/helm/configuration.mdx (hunk 1 above)
Comment (tomhjp) [reply website/content/docs/platform/k8s/helm/configuration.mdx]: Oops, you're right, thanks for catching that

**Pull request merged**